src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -120,6 +120,30 @@ namespace Sapphire::Network::Packets::Server
     PlayerEntry entries[10];
   };
 
+  struct FFXIVIpcExamineSearchInfo : FFXIVIpcBasePacket< ExamineSearchInfo >
+  {
+    uint32_t unknown;
+    uint16_t unknown1;
+    uint16_t unknown2;
+    char padding[16];
+    uint32_t unknown3;
+    uint16_t unknown4;
+    uint16_t unknown5;
+    uint16_t unknown6;
+    uint8_t worldId;
+    char searchMessage[193];
+    char fcName[24];
+    uint8_t unknown7;
+    uint16_t padding1;
+    uint16_t classJobIdLevel[Common::CLASSJOB_SLOTS * 2]; // Pairs of classJobId, classJobLevel
```

Comment:
would need to fix this otherwise size isn't going to be right, but what you could do is just a nested structure

```
struct ClassJobEntry
{
  uint16_t id;
  uint16_t level;
} levelEntries[38];
```
and you can add another constant to common for the number as well

Reply:
oh, that's a better idea than what I just did, 1 sec :v